pkg/controller/common/constants.go

```diff
@@ -1,6 +1,8 @@
 package common
 
 const (
+	MCONamespace = "openshift-machine-config-operator"
```

Comment:
following conventions of the file can we add a comment for this new const?

Reply:
Hmm yeah I always struggle coming up with a non-redundant comment for something like this. Do I have to explain what a namespace is?

Reply:
Maybe `// MCONamespace is the namespace that should be used for all API objects owned by the MCO by default` ?

Reply:
(In Rust¹ it's not required that the documentation comments duplicate the name of the value, which helps avoid redundancy)


¹ yes, yes 😉

Reply:
Added the comment